js/src/forum/components/PostReactAction.js

```diff
@@ -146,6 +146,9 @@ export default class PostReactAction extends Component {
     }
 
     reactButton() {
+        if (this.post.user() === app.session.user) {
+            return ('');
```

Comment:
I don't think the `('')` is necessary here.

```suggestion
            return;
```